posts/2021-11-26-jakarta-ee-9.1.adoc

```diff
@@ -213,6 +210,101 @@ Open Liberty now provides a way to control response headers for a given HTTP end
 
 We’ve spent some time fixing bugs. The following sections describe just some of the issues resolved in this release. If you’re interested, here’s the  link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A21.0.0.12+label%3A%22release+bug%22[full list of bugs fixed in 21.0.0.12].
```

Comment:
link is wrong.  this seems to happen frequently for the release blog posts.... can you update the template so that the link doesn't use `RELEASE_VERSION` and makes it clear that the version is of the X00X format (no periods)